README.md

```diff
@@ -3,7 +3,7 @@
     <img src="https://raw.githubusercontent.com/JoseTomasTocino/freegemas/static/images/header_gems.png"><br>
 </p>
 
-__Freegemas__ is an open source version of the well known Bejeweled, for GNU/Linux, Windows, Mac and the Playstation Vita. It's written in C++ using [SDL2](https://www.libsdl.org/). In the past it used Gosu instead of SDL2, and the old repository can be found at [Google Code](http://freegemas.googlecode.com)
```

Comment:
Since this page no longer exists, I think not changing this link would be better. I'm not sure if it does, but I wouldn't want it to mess with the wayback machine results for it.